sale_procurement_analytic/models/product.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    @api.multi
+    def _need_procurement(self):
+        # When sale/product is installed alone, there is no need to create
```

Comment:
@aheficent No, procurement is not created if only sale/product is installed.

@pedrobaeza  I think it's necessary.

This module has **only** sale and procurement_analytic dependencies.

Without this method, procurement is not ever created at all. Tests fail too. 
If others modules are installed, then procurement is created as you expected.